dockerfiles/scripts/healthcheck-utilities.sh

```diff
@@ -1,10 +1,14 @@
 #
 # Determine whether a local daemon is SYNCed with its network
 #
+"${DAEMON_REST_PORT:=3085}"
 
 function updateSyncStatusLabel() {
     status=$(
-        curl --silent --show-error --header "Content-Type:application/json" -d'{ "query": "query { syncStatus } " }' localhost:3085/graphql | \
+        curl --silent --show-error \
```

Comment:
NIT:  looks like there is a potential to introduce single function and inject query:
```
function sendQraphqlQuery() {
  local __query="$1"

  echo
        curl --silent --show-error \
            --header "Content-Type:application/json" \
            -d'{ "query": "$__query" }' \
            "localhost:${DAEMON_REST_PORT}/graphql"
}
```
and then 
```
function updateSyncStatusLabel() {
    sendQraphqlQuery "query { syncStatus} "  | jq '.data.syncStatus' | sed 's/"//g''
}
```
etc